spec/models/html_document_spec.rb

```diff
@@ -709,6 +709,16 @@
           expect(redirect_url).to eq 'https://www.foo.gov/my%7Curl%E2%80%99s_weird?!'
         end
       end
+
+      context 'when content does not have URL' do
+        let(:raw_document) do
+          '<html><meta http-equiv="refresh" content="43200"></html>'
+        end
+
+        it 'encodes the characters' do
```

Comment:
`encodes the characters` does not describe this spec expectation. However, this expectation is simple enough that we can replace the `it` block with a one-liner:
```ruby:
      context 'when content does not have a URL' do
        let(:raw_document) do
          '<html><meta http-equiv="refresh" content="43200"></html>'
        end

        it { is_expected.to be_nil }
      end
```
Let's also tweak the context for clarity: "when the content parameter does not include a URL"